test/routes/index.spec.js

```diff
@@ -0,0 +1,41 @@
+import assert from 'power-assert';
+import request from 'supertest';
+import session from 'supertest-session';
+import app from 'app.js';
+import UserModel from 'models/User';
+
+describe('/routes/mail', ()=> {
+  let appSession;
+  beforeEach(()=> {
+    appSession = session(app);
+  });
+  it('未ログイン状態ではログインページにリダイレクトされる', (done)=> {
+    request(app).get('/').end((err, ret)=> {
+      assert(ret.status === 303);
+      assert(ret.redirect === true);
+      assert(ret.header.location === '/account/login');
+      done();
+    });
+  });
+
+  it('ログイン状態ではリダイレクトされない', (done)=> {
+    const account = {
+      mail: 'dragonball@jump.com',
+      password: 'wakuwakusitekitazo',
+    };
+    const user = new UserModel(account);
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).